guide/content/helpers/link.haml

```diff
@@ -57,13 +50,14 @@ p
     liens internes ; les liens externes sont déjà marqués par une
     icône donc il est fortement décommandé d'en faire usage sur eux.
 
-  p
-    ' Vous pouvez consulter
-    == link_to('la liste des icônes sur le site du DSFR', 'https://www.systeme-de-design.gouv.fr/elements-d-interface/fondamentaux-techniques/icones/', target: '_blank', rel: 'noopener')
-    | .
+  %p
+    Vous pouvez consulter la
+    = link_to('liste des icônes sur le site du DSFR',
```

Comment:
does this work ? I think you're supposed to append a `|` at the end of multiline ruby strings in haml - including the last one

Reply:
well it does work but I don't understand why, do you?

Reply:
> In addition, sometimes you need to call Ruby methods or declare data structures that just need a lot of template information. So data structures and functions that require lots of arguments can be wrapped over multiple lines, as long as each line but the last ends in a comma (see [Inserting Ruby](https://haml.info/docs/yardoc/file.REFERENCE.html#inserting_ruby)).

oooh nice

Reply:
haml-lint sorted me out, I didn't know either 😄 

https://github.com/sds/haml-lint/blob/main/lib/haml_lint/linter/README.md#multilinepipe